src/ecdsa/ellipticcurve.py

```diff
@@ -1482,6 +1518,28 @@ def __rmul__(self, other):
         """Multiply point by an integer."""
         return self * other
 
+    def _mul_precompute(self, other):
```

Comment:
Is it worth it directly cross-testing this against non-precomputing implementation?

Reply:
it is, and I'm almost sure that I wrote the test to do that, must have forgot to include it, will fix

Reply:
done (`test_ed25519_mul_precompute_vs_naf`)

Reply:
looks OK